lsql/judge/feedback.py

```diff
@@ -50,11 +50,46 @@ def feedback_headers(expected, obtained):
     """
     if expected['header'] == obtained['header']:
         return ''
-    return render_to_string('feedback_wa_headers.html',
-                            {'expected': header_to_str(expected['header']),
-                             'obtained': header_to_str(obtained['header'])}
-                            )
 
+    if len(expected['header']) != len(obtained['header']) and expected['header'] != obtained['header']:
+        _expected = "Esperado: ("+str(len(expected['header']))+" columnas)"
```

Comment:
Concatenar cadenas con el + no mola en Python. Usa la sintaxis de cadenas-f, que es mucho más cómoda y elegante: https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python